docs/Project.toml

```
@@ -1,6 +1,7 @@
 [deps]
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Gadfly = "c91e804a-d5a3-530f-b6f0-dfbca275c004"
```

could we add a compat on Gadfly = "1"

---

you mean edit this file by changing line 4?
```suggestion
Gadfly = "1"
```

---

no not this one, you can add `Gadfly = "1"` in the compat section of the project file

---

Right, thanks